scripts/apply-mail.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+
+import email
+import argparse
+import re
+import tempfile
+import os
+
+argparser = argparse.ArgumentParser()
+argparser.add_argument('emails', metavar='FILE', nargs='+',
+                            help='Files with email message containing a patch')
+
+
+args = argparser.parse_args()
+
+bad_google = re.compile(r'".* via .*" <.*@googlegroups.com>')
+
+temps = []
+
+for email_file in args.emails:
+    e = email.message_from_file(open(email_file))
```

Comment:
I don't think `message_from_file` takes ownership of the file (doesn't close it). The python way of doing this:
```python
with open(email_file) as f:
    e = email.message_from_file(f)
```
You don't have to worry about block scope here (`e` not being available outside `with` block), python doesn't have block scope.

Reply:
It will auto-close when it falls out of scope. message_for_file() will take a reference on the opened file, and as soon as it exits, the file will be closed.